cmd/mesher/mesher.go

```diff
@@ -23,9 +23,6 @@ import (
 	_ "github.com/apache/servicecomb-mesher/proxy/resolver/authority"
 
 	_ "github.com/apache/servicecomb-mesher/proxy/handler"
-	_ "github.com/apache/servicecomb-mesher/proxy/protocol/dubbo/client/chassis"
```

Comment:
Is there a JIRA to track this kind of change? 
What's your plan about the protocol plugins?

Reply:
will raise a issue